ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/LagReportingResource.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.server.resources;
+
+import io.confluent.ksql.rest.entity.ClusterLagsResponse;
+import io.confluent.ksql.rest.entity.LagReportingRequest;
+import io.confluent.ksql.rest.entity.LagReportingResponse;
+import io.confluent.ksql.rest.entity.Versions;
+import io.confluent.ksql.rest.server.LagReportingAgent;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/lag")
```

Comment:
same comment as in vicky's PR.. should we just consolidate all of this into the cluster status resource.. atleast the active/standby info and lag info for the same partitions reported on that endpoint, can be consolidated?

Reply:
Done.  Moved the lag info to that endpoint.

Reporting still belongs in its own resource though, so I left that here.